plugins/generator-1.15.2/forge-1.15.2/procedures/entity_ischild.java.ftl

```diff
@@ -0,0 +1 @@
+(((LivingEntity) ${input$entity}).isChild())
```

Comment:
You can't directly cast to LivingEntity as there can be cases where this block is not used with LivingEntity. Check how for example https://github.com/MCreator/MCreator/blob/master/plugins/generator-1.15.2/forge-1.15.2/procedures/entity_armor_value.java.ftl does it and do similarly yourself, returning false if not the right type.

Reply:
It's done, I've changed the code. ;)